src/api/utils.py

```diff
@@ -24,3 +24,12 @@ def update(model, attr_name, attr_value):
         and attr_value != getattr(model, attr_name)
     ):
         setattr(model, attr_name, attr_value)
+
+def modify_attribute(model, attr_name, attr_value):
```

Comment:
this should be unnecessary to use and you can just modify the attribute directly

Reply:
it wouldn't let me access it directly, but i saw that we already had another update function that does what modify_attribute did